README.md

```diff
@@ -17,50 +18,140 @@ NPM](https://github.com/apiaryio/drafter-npm) package instead of Protagonist
 directly as Protagonist needs to be compiled which may not be possible in every
 situation*
 
-The best way to install Protagonist is by using its [NPM package](https://npmjs.org/package/protagonist).
+Protagonist can be installed via the [Protagonist npm
+package](https://npmjs.org/package/protagonist) by
+[npm](https://docs.npmjs.com/cli/npm) or [yarn](https://yarnpkg.com/en/).
 
 ```sh
 $ npm install protagonist
+# or
+$ yarn install protagonist
 ```
 
-**NOTE:** *Installing Protagonist depends on having Python 2 installed along with a compiler such as GCC, Clang or MSVC.*
+Protagonist uses the
+[node-gyp](https://github.com/nodejs/node-gyp#installation) built tool which
+requires Python 2.7 (3.x is not supported) along with a compiler and other
```

Comment:
linking to https://stackoverflow.com/a/22433804/325365 ?

Reply:
There's a link to https://github.com/nodejs/node-gyp#installation which goes into detail about that (https://github.com/nodejs/node-gyp#configuring-python-dependency), there's also similiar about configuring visual studio.